### PR TITLE
Tailwind

### PR DESCRIPTION
**Pull request by m1guelpf**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

- Tailwind is a new utility-based CSS framework that has been out less than a month, but the community alround it and the things you can do without even tweaking the configuration are awesome.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/tailwind/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: tailwind-css, tailwindcss
```
Comment (cheshire137) [inline topics/tailwind/index.md]: You could also add tailwind-scaffolding because it's in use: https://github.com/topics/tailwind-scaffolding.

File: topics/tailwind/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+created_by: Adam Wathan, Jonathan Reinink, David Hemphill, Steve Schoger
+display_name: Tailwind CSS
+github_url: https://github.com/tailwindcss
+logo: tailwind.png
+related: framework, css, postcss
+released: October 2017
+short_description: Tailwind is a Utility-First CSS framework.
```
Comment (cheshire137) [inline topics/tailwind/index.md]: Can you lowercase "Utility-First"?

File: topics/tailwind/index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+topic: tailwind
+url: https://tailwindcss.com/
+---
+A Utility-First CSS Framework for Rapid UI Development
```
Comment (cheshire137) [inline topics/tailwind/index.md]: Could you lowercase this and add a period?

Can this be expanded? It should give more information than what the `short_description` does. For example: why would I choose Tailwind over Bootstrap/Bulma/Materialize? Is Tailwind particularly smaller than other CSS frameworks? Does it support mobile? Does it support older browsers? Etc.

File: topics/tailwind/index.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: tailwind-css, tailwindcss
+created_by: Adam Wathan, Jonathan Reinink, David Hemphill, Steve Schoger
+display_name: Tailwind CSS
+github_url: https://github.com/tailwindcss
+logo: tailwind.png
+related: framework, css, postcss
```
Comment (cheshire137) [inline topics/tailwind/index.md]: You could add `css-framework`.

**Comment by m1guelpf** [comment]
@cheshire137 Done!

File: topics/tailwind/index.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+display_name: Tailwind CSS
+github_url: https://github.com/tailwindcss
+logo: tailwind.png
+related: framework, css, postcss, css-framework
+released: October 2017
+short_description: Tailwind is a Utility-First CSS framework.
+topic: tailwind
+url: https://tailwindcss.com/
+---
+A utility-first CSS framework for rapid UI development written in PostCSS and configured through JavaScript, which means it’s highly customizable by design. It doesn’t have a default theme and nor built-in UI components but this also means it has no opinion about how your site should look and doesn’t impose design decisions that you have to fight to undo.
```
Comment (cheshire137) [inline topics/tailwind/index.md]: Can you rephrase the first part, maybe just by adding "Tailwind is" in front of "a utility-first CSS framework..."?

Maybe instead of "which means it’s highly customizable", "making it highly customizable"?

Maybe for the second sentence:

> Tailwind doesn’t have a default theme nor built-in UI components. This means it has no opinion about how your site should look and doesn’t impose design decisions that can be difficult to undo.

**Comment by m1guelpf** [comment]
@cheshire137 Done!

File: topics/tailwind/index.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: tailwind-css, tailwindcss, tailwind-scaffolding
+created_by: Adam Wathan, Jonathan Reinink, David Hemphill, Steve Schoger
+display_name: Tailwind CSS
+github_url: https://github.com/tailwindcss
+logo: tailwind.png
+related: framework, css, postcss, css-framework
+released: October 2017
+short_description: Tailwind is a Utility-First CSS framework.
```
Comment (cheshire137) [inline topics/tailwind/index.md]: Could you lowercase "Utility-First" here?

**Comment by m1guelpf** [comment]
@cheshire137 Done!
